cmd/wait/install/cmd.go

```diff
@@ -0,0 +1,159 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package install
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift/rosa/pkg/aws"
+	"github.com/openshift/rosa/pkg/logging"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/output"
+	rprtr "github.com/openshift/rosa/pkg/reporter"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "install",
+	Short: "Waits for install to finish",
+	Long:  "Waits for install to finish",
+	Run:   run,
+}
+
+var args struct {
+	clusterKey   string
+	pollInterval int
+	maxInterval  int
+}
+
+func init() {
+	flags := Cmd.Flags()
+
+	output.AddFlag(Cmd)
+
+	flags.StringVarP(
+		&args.clusterKey,
+		"cluster",
+		"c",
+		"",
+		"Name or ID of the cluster to describe.",
+	)
```

Comment:
We now have an `ocm.AddClusterFlag(Cmd)` helper function that adds validation and auto-complete for cluster names. Then belog, instead of `clusterKey := args.clusterKey` and `ocm.IsValidClusterKey()`, you can call `clusterKey, err := ocm.GetClusterKey()` instead

Reply:
Truly helpful, done!